features/karaf-health/src/main/java/org/opennms/features/karaf/health/bundle/Activator.java

```diff
@@ -26,25 +26,32 @@
  *     http://www.opennms.com/
  *******************************************************************************/
 
-package org.opennms.features.vaadin.components.bundlerefresher;
+package org.opennms.features.karaf.health.bundle;
+
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Dictionary;
+import java.util.Hashtable;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.opennms.features.karaf.health.service.KarafHealthService;
 import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleActivator;
 import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
 import org.osgi.framework.wiring.FrameworkWiring;
 
-
 public class Activator implements BundleActivator {
 
+    private ServiceRegistration karafHealthServiceSvcReg;
+
     @Override
     public void start(BundleContext context) throws Exception {
-        // Determine if any vaadin theme fragments are unresoled
+        // Determine if any vaadin theme fragments are unresolved
```

Comment:
@mvrueden is it work keeping the Vaadin bundle refreshing parts around? Or do you think it's safe to remove them now?

Reply:
I am not 100% sure. We talked about removing it, as Karaf added a fix for this at some point. However we have never actually verified it. I suggest we create an issue to remove the bundle refresher code and leave it for now. After this is merged we can then focus on removing the bits (or verify if they can remove and then remove it)